src/drivers/uavcan/uavcan_drivers/kinetis/driver/include/uavcan_kinetis/can.hpp

```diff
@@ -360,18 +360,13 @@ class CanInitHelper
 	 * This function can either initialize the driver at a fixed bit rate, or it can perform
 	 * automatic bit rate detection. For theory please refer to the CiA application note #801.
 	 *
-	 * @param delay_callable    A callable entity that suspends execution for strictly more than one second.
```

Comment:
Same here. The two commits in this PR, cherry pick them fully and make a separate PR of that. When it is merged, then make another PR of the 3 remaining commits.